fenix/app/src/main/java/org/mozilla/fenix/debugsettings/navigation/DebugDrawerRoute.kt

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.debugsettings.navigation
+
+import androidx.annotation.StringRes
+import org.mozilla.fenix.R
+import org.mozilla.fenix.debugsettings.tabs.TabTools
+
+/**
+ * The navigation routes for screens within the Debug Drawer.
+ *
+ * @property route The navigation route of the destination.
```

Comment:
In our direct use case, it is. The navigation library can technically take a URI as its navigation input, `route`. The route can be just a simple string, but it can also have a data model in it. It can also support deep linking. So I'm calling it `route` to keep the naming in sync with the library, but also not confuse it any of our data model abstractions.

Reply:
👍 
```
The navigation route of the destination.
``` 
Can we do something to clarify what a `route` is, perhaps including what you've mentioned here around the URI?

Reply:
Sure thing! I'll look into updating the docs with `route` in the plumbing PR.